test/app/presenters/track_test.rb

```diff
@@ -6,21 +6,34 @@
 class PresentersTrackTest < Minitest::Test
   def setup
     @track = ExercismWeb::Presenters::Track.new("test_track")
+
+    trackler_tracks = Object.new
     @trackler_track = Object.new
-    @track.stubs(:trackler_track).returns(@trackler_track)
-    @track.stubs(:fallback_topic_content).returns("Default topic content")
+
+    Trackler.stubs(:tracks).returns(trackler_tracks)
+    trackler_tracks.stubs(:[]).returns(@trackler_track)
+    @trackler_track.stubs(:repository).returns("https://github.com/exercism/test_track")
+    @trackler_track.stubs(:doc_format).returns("md")
   end
 
   def test_docs_with_trackler_content
-    trackler_docs = OpenStruct.new(test_topic: "Track-specific content from Trackler")
-    @trackler_track.stubs(:docs).returns(trackler_docs)
-    assert_equal @track.docs["test_topic"], "Track-specific content from Trackler"
+    @trackler_track.stubs(:docs).returns(OpenStruct.new(about: "Track-specific content from Trackler"))
+    topic_doc = @track.docs["about"]
+
+    assert topic_doc.include? "Track-specific content from Trackler"
+    assert topic_doc.include? "Help us explain this better!"
+    assert topic_doc.include? "File a GitHub issue at https://github.com/exercism/test_track"
+    assert topic_doc.include? "https://github.com/exercism/test_track/blob/master/docs/ABOUT.md"
   end
 
   def test_docs_without_trackler_content
-    trackler_docs = OpenStruct.new(test_topic: "")
-    @trackler_track.stubs(:docs).returns(trackler_docs)
-    assert_equal @track.docs["test_topic"], "Default topic content"
+    @trackler_track.stubs(:docs).returns(OpenStruct.new(about: ""))
+    topic_doc = @track.docs["about"]
+
+    assert topic_doc.include? "We're missing a short introduction about the language."
+    refute topic_doc.include? "Help us explain this better!"
+    refute topic_doc.include? "File a GitHub issue at https://github.com/exercism/test_track"
+    refute topic_doc.include? "https://github.com/exercism/test_track/blob/master/docs/TEST_TOPIC.md"
```

Comment:
Good decision.
Refactoring the tests after you've written the code is an important and often neglected step in TDD.